cpp/src/arrow/compute/function_test.cc

```diff
@@ -323,7 +323,7 @@ TEST(ScalarAggregateFunction, DispatchExact) {
 
   std::vector<InputType> in_args = {int8()};
   ScalarAggregateKernel kernel(std::move(in_args), int64(), NoopInit, NoopConsume,
-                               NoopMerge, NoopFinalize);
+                               NoopMerge, NoopFinalize, /*ordered*/ false);
```

Comment:
```suggestion
                               NoopMerge, NoopFinalize, /*ordered=*/ false);
```
Minor consistency nit